### ISSUE TITLE
[bug] Next.js + TypeScript + SWC Minification

### ISSUE THREAD

**Issue by giliamverheide** [rainbow-me/rainbowkit#383]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### RainbowKit Version

0.1.0

### wagmi Version

0.3.5

### Current Behavior

Using Next.js, TypeScript, and minification with the SWC compiler, causes rainbowkit to break. Happens with both a static site export and building and running a next server. I'm unsure if this is something that can be fixed inside the rainbowkit package or if this is an issue within `swc`.

In short: typescript + the `swcMinify` option inside `next.config.js` causes this issue.

### Expected Behavior

Work with the swc compiler.

### Steps To Reproduce

Please refer to the linked repository for a minimal reproducible example and steps to reproduce.

### Link to Minimal Reproducible Example (CodeSandbox, StackBlitz, etc.)

https://github.com/giliamverheide/next-ts-swc-rainbowkit-issue

**Comment by giliamverheide** [comment]
Update: This should get fixed in https://github.com/vercel/next.js/pull/36745

**Comment by nickbytes** [comment]
Thanks for the update @giliamverheide – going to close this issue! Feel free to re-open if you think there's something else going on here w/ RainbowKit!
